tests/test_binaries.py

```diff
@@ -318,14 +325,23 @@ def test_eth_regressions(self):
                 in_directory=issue.get('in_directory')
             )
 
+    def test_eth_1102(self):
+        with tempfile.TemporaryDirectory() as workspace:
+            self._simple_cli_run('1102.sol', args=['--detect-overflow'], workspace=workspace)
+
+            with open(os.path.join(workspace, 'global.findings')) as gf:
+                global_findings = gf.read().splitlines()
+
+        self.assertEqual(global_findings[0], '- Unsigned integer overflow at SUB instruction -')
+        self.assertRegex(global_findings[1], '  Contract: 0x:\d+  EVM Program counter: 0x175')
+        self.assertEqual(global_findings[2], '  Solidity snippet:')
+        self.assertEqual(global_findings[3], '    10  count -= input')
+        self.assertEqual(global_findings[4], '')
+
+        self.assertEqual(len(global_findings), 5)
+
     def test_eth_705(self):
-        # This test needs to run inside tests/binaries because the contract imports a file
-        # that is in the tests/binaries dir
-        old_cwd = os.getcwd()
```

Comment:
This code didn't change the cwd at all before running Manticore.

Also I believe that recent changes (that made manticore work fine with contracts that use imports) makes this redundant.

After removing this part, the test passed locally, let's see if it works on Travis.